tusclient/client.py

```diff
@@ -25,6 +25,20 @@ class TusClient:
     def __init__(self, url: str, headers: Optional[Dict[str, str]] = None):
         self.url = url
         self.headers = headers or {}
+        self.client_cert = None
+
+    def set_certificate(self, cert: Optional[Tuple[str, str]] = None):
```

Comment:
Maybe pass the cert in init and don't provide this function?

=> I think I'll update the PR to take the client certificate in __init__ only. Creating a new uploader with different certificate looks better imho (url cannot be updated either)

Reply:
Moved to __init__. Leaving open for now.